adabot/arduino_libraries.py

```diff
@@ -162,6 +172,27 @@ def validate_release_state(repo):
 
     return
 
+def validate_travis(repo):
+    """Validate if a repo has .travis.yml.
+    """
+    repo_has_travis = github.get("/repos/" + repo["full_name"] + "/contents/.travis.yml")
+    if repo_has_travis.ok:
+        return True
+
+def has_ino(repo):
+    """Validate if a repo has any *.ino files.
+    """
+    repo_has_ino = github.get("/search/code?q=extension:ino+repo:adafruit/" + repo["name"])
```

Comment:
If you want to avoid hitting the `search` rate limit, using the `/repos/<repo_name>/contents` endpoint is helpful.

CircuitPython equivalent has an example: https://github.com/adafruit/adabot/blob/master/adabot/lib/circuitpython_library_validators.py#L428